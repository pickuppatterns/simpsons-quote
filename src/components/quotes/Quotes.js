import React from 'react';
import PropTypes from 'prop-types';
import { css } from 'emotion';



function Quotes({ quote, characterImg, characterName }) {
  return (
    <>
      <section 
        className={css`
        overflow: auto;
        font-family: helvetica;
        width: 100%;
    `}>
        <h2>{characterName}</h2>
        <p>{quote}</p>
        <img src={characterImg} />
      </section>
    </> 
  );
}

Quotes.propTypes = {
  quote: PropTypes.string.isRequired,
  characterImg: PropTypes.string.isRequired,
  characterName: PropTypes.string.isRequired
};
export default Quotes;

