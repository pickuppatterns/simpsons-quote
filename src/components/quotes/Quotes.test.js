import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';
import { MemoryRouter } from 'react-router-dom';

describe('Quotes', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Quotes />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

