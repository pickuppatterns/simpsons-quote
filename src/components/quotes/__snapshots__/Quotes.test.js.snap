// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches a snapshot 1`] = `
<section
  className="css-2bwpvc"
>
  <h2 />
  <p />
  <img />
</section>
`;

exports[`Quotes matches a snapshot 1`] = `
<section
  className="css-2bwpvc"
>
  <h2 />
  <p />
  <img />
</section>
`;
